src/main/java/seedu/exercise/model/property/Date.java

```diff
@@ -41,6 +44,105 @@ public static boolean isValidDate(String test) {
         }
     }
 
+    /**
+     * Returns true if given end date is before given start date.
+     */
+    public static boolean isEndDateAfterStartDate(String startDate, String endDate) {
+        try {
+            LocalDate sDate = LocalDate.parse(startDate, formatter);
+            LocalDate eDate = LocalDate.parse(endDate, formatter);
+            if (eDate.compareTo(sDate) < 0) {
+                return false;
+            } else {
+                return true;
+            }
+        } catch (DateTimeParseException e) {
+            return false;
+        }
+    }
+
+    /**
+     * Returns true if given date is between start date and end date.
+     */
+    public static boolean isBetweenStartAndEndDate(Date date, Date startDate, Date endDate) {
+        LocalDate sDate;
+        LocalDate eDate;
+        LocalDate d;
+
+        try {
+            d = LocalDate.parse(date.toString(), formatter);
+            sDate = LocalDate.parse(startDate.toString(), formatter);
+            eDate = LocalDate.parse(endDate.toString(), formatter);
+        } catch (DateTimeParseException e) {
+            return false;
+        }
+
+        if (d.compareTo(sDate) >= 0 && d.compareTo(eDate) <= 0) {
+            return true;
+        }
+
+        return false;
+    }
+
+    /**
+     * Returns the number of days between start date and end date.
+     */
+    public static int numberOfDaysBetween(Date startDate, Date endDate) {
+        LocalDate sDate;
+        LocalDate eDate;
+        try {
+            sDate = LocalDate.parse(startDate.toString(), formatter);
+            eDate = LocalDate.parse(endDate.toString(), formatter);
+        } catch (DateTimeParseException e) {
+            return -1;
+        }
+
+        return (int) DAYS.between(sDate, eDate);
+    }
+
+    /**
+     * Returns today's Date.
+     */
+    public static Date getToday() {
+        LocalDate today = LocalDate.now(ZoneId.of("Asia/Singapore"));
```

Comment:
you may want to use `ZoneId.systemDefault()` instead of hardcoding our zone

Reply:
done